redash/query_runner/impala_ds.py

```diff
@@ -56,6 +56,7 @@ def configuration_schema(cls):
                 "ldap_user": {"type": "string"},
                 "ldap_password": {"type": "string"},
                 "timeout": {"type": "number"},
+                "user": {"type": "string"},
```

Comment:
I wonder how we can communicate to users when to use `user` versus `ldap_user`?

Reply:
I thought that 'ldap_user' and 'user' should work separately, maybe I should add this condition test